httpql/src/main/java/com/hubspot/httpql/DefaultMetaUtils.java

```diff
@@ -31,6 +33,24 @@ public static FilterJoin findFilterJoin(BeanPropertyDefinition prop) {
     return findAnnotation(prop, FilterJoin.class);
   }
 
+  @Nullable
+  public static FilterJoinByDescriptor findFilterJoinByDescriptor(BeanPropertyDefinition prop) {
```

Comment:
`getFilter*`?

Reply:
Following the pattern already established above.